test/pageObjects/admin/profile.js

```diff
@@ -0,0 +1,62 @@
+class Profile {
+
+  get profileNavbar() {
+    return $('[data-testid="navbar"] li:nth-child(3) a');
+  }
+
+  get profileForm() {
+    return $('[data-testid="admin-profile-form"]');
+  }
+
+  get inputName() {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(2) input:nth-child(2)');
+  }
+
+  get inputLastName() {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(3) input:nth-child(2)');
+  }
+
+  get inputDni () {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(4) input:nth-child(2)');
+  }
+
+  get inputPhone() {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(5) input:nth-child(2)');
+  }
+
+  get inputEmail() {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(6) input:nth-child(2)');
+  }
+
+  get inputCity() {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(7) input:nth-child(2)');
+  }
+
+  get inputPassword() {
+    return $('[data-testid="admin-profile-form"] fieldset:nth-child(8) input:nth-child(2)');
+  }
+
+  get path (){
+    return $('#root > div > div > div > div > form > div:nth-child(1) > fieldset:nth-child(8) > div > svg > path');
```

Comment:
you could change this to use the svg inside the form with test id admin-profile-form